Testing/SystemTests/tests/framework/ISIS_PowderPolarisTest.py

```diff
@@ -672,7 +672,9 @@ def setup_inst_object(mode, with_container=False):
         )
 
     sample_details = SampleDetails(height=4.0, radius=0.2985, center=[0, 0, 0], shape="cylinder")
-    sample_details.set_material(chemical_formula="Si")
+    # define multi-atom cell to stop regression of bug calling MaterialBuilder.build() without setting number density
+    # which works for one atom cells as number density is automatically calculated
+    sample_details.set_material(chemical_formula="Si Si", number_density=0.04996)
```

Comment:
No, users will set the maerial as/how they always have, here I specify the chemical formula as `"Si Si"` because mantid has a lookup somewhere in `MaterialBuilder` to find the number density if you specify a single atom formula (e.g `"Si"`) which meant the tests weren't catching the bug found by POLARIS scientists (with real samples containing more atom). The two formula `"Si Si"` and `"Si"` should be equivalent as the default unit for number density in mantid is (for better or worse) atoms/Ang^3

Reply:
Ah I see, thanks for clarifying!